crates/js-sys/src/lib.rs

```diff
@@ -1596,7 +1596,7 @@ extern "C" {
     ///
     /// [MDN documentation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/DataView)
     #[wasm_bindgen(constructor)]
-    pub fn new(buffer: &ArrayBuffer, byteOffset: usize, byteLength: usize) -> DataView;
+    pub fn new(buffer: &JsValue, byteOffset: usize, byteLength: usize) -> DataView;
```

Comment:
AFAIK this isn't [officially](https://doc.rust-lang.org/1.73.0/cargo/reference/semver.html) considered a breaking change, but it can still actually break code.

Though I believe I would be fine with that, I think it's better to just introduce another constructor function, like we do in other APIs.

Reply:
No problem. As this is my first contribution to this project I'm unfamiliar w/ naming conventions, etc. Where should I look for a good example of where something similar was done so I can maintain consistency?